src/audiovideofacade/DefaultAudioVideoFacade.ts

```diff
@@ -251,9 +251,18 @@ export default class DefaultAudioVideoFacade implements AudioVideoFacade {
     this.trace('realtimeSubscribeToVolumeIndicator', attendeeId);
   }
 
-  realtimeUnsubscribeFromVolumeIndicator(attendeeId: string): void {
-    this.realtimeController.realtimeUnsubscribeFromVolumeIndicator(attendeeId);
-    this.trace('realtimeUnsubscribeFromVolumeIndicator', attendeeId);
+  realtimeUnsubscribeFromVolumeIndicator(
+    attendeeId: string,
+    callback?: (
+      attendeeId: string,
+      volume: number | null,
+      muted: boolean | null,
+      signalStrength: number | null,
+      externalUserId?: string
+    ) => void
```

Comment:
Suggest naming and sharing this type: `VolumeIndicatorCallback`, perhaps.

Reply:
If you end up putting this in its own file, use `import type` to import that file.

Reply:
Done!